cmake/FindJansson.cmake

```diff
@@ -0,0 +1,15 @@
+if (Jansson_INCLUDE_DIRS AND Jansson_LIBRARY AND Jansson_VERSION)
+  set(Jansson_FOUND TRUE)
+else ()
+  find_path(Jansson_INCLUDE_DIRS NAMES jansson.h)
```

Comment:
Why is this manual search approach necessary? What is that case?

Reply:
To the best of my knowledge, neither Jansson nor CMake implement any functionality for searching and setting `Jansson_VERSION` that we would eventually need in the `find_package_or_fallback_to_3party` macro.

Reply:
Interesting, I see, packages have only .h and .a/.so in them.

If we keep all 3ps up to date, then the chance is low that Linux distribution will also quickly update to the latest version. It means that on most systems our 3party folder will be used anyway, or OM can't be installed because of unsatisfied dependencies versions...

Reply:
The likes of `brew` and `tea` install the latest on macOS and Linux all the time.
There are lots of rolling Linux distros which also deliver the latest version pretty much all the time, and there are "distros" where the user builds the latest from source anyway. And there are completely ordinary mainstream disitros like Ubuntu Kinetic, which happen to deliver the latest at the time of writing this.